doc/content/tutorials/efas/introduction/concluding_remarks.md

```diff
@@ -0,0 +1,54 @@
+
+The electric potential on the toolset steadily decreases from the top of the toolset to the bottom of it, as observed in the contour plot below. This indicates that the direction of electric current is from top to bottom, and this current direction corresponds with how the DCS-5 machine is constructed. 
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/potential_temp_screenshot.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=potential-temp-screenshot
+    caption=Electrical Potential and Temperature Contour Plots.
+
+Electric field values slightly between different wall thickness, but noticeably changes on the graphite spacers. The electric field is highly concentrated in the center of these graphite spacers and seems to correspond with the higher temperature values in the same spot. This phenomenon is shown in [10mm-elec-field-and-temp], for example. However, the powder casing and die wall do not experience a similar spike in temperature or electric potential to the graphite spacers and may need to be considered when analyzing an EFAS toolset.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/5mm_elec_field_and_temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=5mm-elec-field-and-temp
+    caption=Electric Field and Temperature Contour Plots for a 5 mm wall thickness at t = 954 seconds.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/10mm_elec_field_and_temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=10mm-elec-field-and-temp
+    caption=Electric Field and Temperature Contour Plots for a 10 mm wall thickness at t = 960 seconds.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/13.875mm_elec_field_and_temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=13.875mm-elec-field-and-temp
+    caption=Electric Field and Temperature Contour Plots for a 13.875 mm wall thickness at t = 960 seconds.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/20mm-elec-field-and-temp.png
+    style=width:80%;margin-left:auto;margin-right:auto;
+    id=20mm-elec-field-and-temp
+    caption=Electric Field and Temperature Contour Plots for a 20 mm wall thickness at t = 966 seconds.
+
+!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/25mm-elec-field-and-temp.png
```

Comment:
try leaving this file path name with the underscores to match the file name you created in a previous commit:
```suggestion
!media media/tutorial_dcs5_images/tutorial_pngs/results_pngs/25mm_elec_field_and_temp.png
```